libraries/jevmodal/media/css/jevmodal.css

```diff
@@ -161,7 +162,7 @@
 	max-height:100%;
 }
 .jevmodal .modal-dialog {
-    height: 100%;
+    height: 95%;
```

Comment:
Please send me the details and I'll take a look.  Did you implement 
these changes there already?

Geraint

On 14/08/2015 16:56, Carlos (carcam) wrote:
>
> In libraries/jevmodal/media/css/jevmodal.css 
> <https://github.com/JEvents/JEvents/pull/289#discussion_r37090478>:
>
> > @@ -161,7 +162,7 @@
> >  	max-height:100%;
> >  }
> >  .jevmodal .modal-dialog {
> > -    height: 100%;
> > +    height: 95%;
>
> It's indeed! The site is Kenneth's testing site, let me know if you 
> need site access.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/JEvents/JEvents/pull/289/files#r37090478>.
>